### PR TITLE
Ignore unreliable test

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

File: querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java [hunk 1]
```diff
@@ -93,10 +94,14 @@ public void list_distinct() {
 
     @Test
     public void list_distinct_two_sources() {
-        // XXX List implementation of JDO provider has weird equals implementation
-        assertEquals(
-                ImmutableList.copyOf(query().from(product, product2).select(product, product2).fetch()),
-                ImmutableList.copyOf(query().from(product, product2).distinct().select(product, product2).fetch()));
+        try {
+            // XXX List implementation of JDO provider has weird equals implementation
```
Comment (johnktims) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: What is the `XXX` for?

Comment (Shredder121) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: Then it shows up in a list of TODOs, HACKs, BUGs, XXXs, etc

**Comment by Shredder121** [comment]
This looks fine to me now.

File: querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java (hunk 1 above)
Comment (Shredder121) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: I myself use `XXX` for unclassifiable action items, not a `BUG` or `HACK`.

Comment (timowest) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: Ideally we should abstract that away in Querydsl, so it's kind of an action item

Comment (Shredder121) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: Do you think so? Do you mean copying it over into a new list?
I don't quite know if it's worth the overhead, personally.
What's your view on this?

Comment (timowest) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: It's a bug in the underlying JDO implementation, and we cover lots of Hibernate bugs, so users might also expect that we cover this one.

Comment (Shredder121) [inline querydsl-jdo/src/test/java/com/querydsl/jdo/BasicsTest.java]: Okay, I get you.

**Comment by johnktims** [comment]
About time!

**Pull request merged**